openupgrade_scripts/scripts/survey/14.0.3.2/upgrade_analysis.txt

```diff
@@ -91,6 +91,14 @@ DEL ir.model.access: survey.access_survey_label_all
 DEL ir.model.access: survey.access_survey_label_survey_manager
 DEL ir.model.access: survey.access_survey_label_survey_user
 DEL ir.model.access: survey.access_survey_label_user
+NEW ir.model.constraint: survey.constraint_survey_question_positive_answer_score
+NEW ir.model.constraint: survey.constraint_survey_question_scored_date_have_answers
+NEW ir.model.constraint: survey.constraint_survey_question_scored_datetime_have_answers
+NEW ir.model.constraint: survey.constraint_survey_survey_certification_check
+NEW ir.model.constraint: survey.constraint_survey_survey_scoring_success_min_check
+NEW ir.model.constraint: survey.constraint_survey_survey_session_code_unique
+ir.model.constraint: survey.constraint_survey_user_input_unique_token (changed definition: is now 'unique(access_token)' ('unique(token)'))
+DEL ir.model.constraint: survey.constraint_survey_survey_certificate_check
```

Comment:
I think we need to remove this constraint manually in the `survey` migration scripts.

Reply:
PR of `survey` is not merged. I will update that PR when this one is merged.